controllers/control-plane/kyma_remote_sync_test.go

```diff
@@ -272,6 +183,50 @@ var _ = Describe("Kyma sync into Remote Cluster", Ordered, func() {
 			Should(Succeed())
 	})
 
+	It("Enable Custom ModuleTemplate in SKR", func() {
```

Comment:
To reviewer:

This test combines "Kyma with remote module templates" test and is refactored according to real scenario, in the original test, it enabled `moduleInKcp` in KCP kyma, which did not meet the actual case, although the test works.

```
kyma.Spec.Modules = []v1beta2.Module{moduleInSkr, moduleInKcp}
```